crates/adapters/src/server/mod.rs

```diff
@@ -247,6 +247,7 @@ where
     .shutdown_timeout(10)
     .listen(listener)
     .map_err(|e| ControllerError::io_error("binding server to the listener".to_string(), e))?
+    .workers(2)
```

Comment:
The server also handles HTTP ingress/egress.  I don't have a good feel for how much CPU that needs, but at millions of events/s could be quite a bit. 128 is certainly too much. I wonder if 2 is too little though. To pull a number out of thin air, maybe 8 or 16 is a better default.

Reply:
I dropped it.